resources/build/build-utils.md

```diff
@@ -339,6 +360,29 @@ this, you should use `--debug,-d` to enable the shorthand form.
 Note that you should not include any of the [standard builder parameters] here.
 
 
+**Dependencies** are defined with a `@` prefix, for example:
+
+```bash
+builder_describe "Sample script" \
+  "@/core configure build" \
+  configure \
+  build
+```
+
+A dependency always starts with `@`. The path to the dependency will be relative
+to the build script folder, or to the root of the repository, if the path starts
+with `/`, not to the root of the file system. It is an error to specify a
+dependency outside the repo root.
```

Comment:
This is somewhat unclear as written; it was difficult for me to recognize what part the "if the path starts with `/`" attaches to due to the complexity of the second sentence.

Just a suggestion, but I feel the following structure to be clearer.

```suggestion
A dependency always starts with `@`. The path to the dependency will be relative
to the build script folder if the path does not start with `/`.  Otherwise, the path
to the dependency is interpreted relative to the root of the repository. It is an 
error to specify a dependency outside the repo root.
```